retropie_packages.sh

```diff
@@ -73,5 +73,5 @@ fi
 printMsgs "console" "${__INFMSGS[@]}"
 
 if [[ -n $__joy2key_pid ]]; then
-    kill -INT $__joy2key_pid 2>/dev/null
+    kill -INT $__joy2key_pid 2>/dev/null || echo "Failed to kill joy2key"
```

Comment:
then the correct fix would be to add an exit 0 at the end of the code if the kill if this is the cause, but there are lots of places where non 0 return codes do not mean an error, eg with grep etc. I'm happy to add an exit 0 at the end if it will help.